states/alabama/elections.toml

```diff
@@ -3,7 +3,60 @@ date = 2020-07-14
 name = "Congressional Primary Run-Off (if needed)"
 original_date = 2020-07-14
 
+[20200714.registration]
+received_by = 2020-06-29
+sources = ["https://www.sos.alabama.gov/sites/default/files/2020%20Voter%20Guide%205-15-2020.pdf",
+           "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action"]
+
+[20200714.poll]
+in_person_by = 2020-07-14T19:00:00
```

Comment:
Can add `in_person_start` as well since polls open at 7am.

Reply:
This is grand! I was wondering if there was a field for that since I was able to find the start and end times!